project/LinkerdBuild.scala

```diff
@@ -65,7 +65,8 @@ object LinkerdBuild extends Base {
       .settings(coverageExcludedPackages := ".*XXX_.*")
 
     val baseHttp = projectDir("router/base-http")
-      .dependsOn(core)
+      .dependsOn(core, Finagle.h2)
```

Comment:
I think it would be nice to avoid depending on http and H2 in http-base if possible